tests/python-pytest/onnx/import/test_cases.py

```diff
@@ -85,7 +85,8 @@
     'test_operator_exp',
     'test_operator_maxpool',
     'test_operator_params',
-    'test_operator_permute2'
+    'test_operator_permute2',
+    'test_depthtospace'
```

Comment:
no test for SpaceToDepth in onnx?

Reply:
The test is in review. Hasn't been merged into ONNX yet.

Reply:
Added tests for SpaceToDepth